consensus/poet/core/sawtooth_poet/poet_consensus/poet_fork_resolver.py

```diff
@@ -109,27 +109,43 @@ def compare_forks(self, cur_fork_head, new_fork_head):
                 block=new_fork_head,
                 poet_enclave_module=poet_enclave_module)
 
+
         # If we ever get a new fork head that is not a PoET block, then bail
         # out.  This should never happen, but defensively protect against it.
         if new_fork_wait_certificate is None:
+            print("new_fork_wait_certificate is ", new_fork_wait_certificate)
```

Comment:
These should be `LOGGER.debug` statements, or removed if they don't warrant that.